src/components/TestDetailsDialog/TestDetailsModal.tsx

```diff
@@ -481,6 +487,22 @@ const TestDetailsModal: React.FunctionComponent<{
         onOriginalSizeClick={setOriginalSize}
         onFitIntoScreenClick={fitStageToScreen}
       />
+      <BaseModal
+        open={applyIgnoreDialogOpen}
+        title={''}
```

Comment:
you could reuse text from tooltip 
line 362`Apply selected ignore area to all images in this build.`